packages/nimble-components/src/card/tests/card.stories.ts

```diff
@@ -37,9 +42,18 @@ const metadata: Meta = {
                 <${listOptionTag} value="3">Option 3</${listOptionTag}>
             </${selectTag}>
         </${cardTag}>
-    `)
+    `),
+    argTypes: {
+        title: {
+            description:
+                'Text displayed as a title inside the card. Cards should **always include a title**. The title is used to provide an accessible name to assistive technologies.<br><br>Provide the title in an `inline` element such as `<span>` that is targeted to the `title` slot.'
```

Comment:
> Provide the title in an `inline` element such as `<span>` that is targeted to the `title` slot.

The places where we give this guidance today have style dependencies on `inline` elements being slotted (e.g. messes up ellipsis if a block element is slotted for the title). Right now, I don't believe that's the case for the title of the card. I lean towards removing that statement, at least for now.

Reply:
Thanks for pointing this out. I shortened this sentence to: "Provide the title in an element targeted to the `title` slot."